content/en/docs/instrumentation/java/manual.md

```diff
@@ -35,6 +35,87 @@ the most important piece of telemetry source-identifying info.
 
 For example:
 
+<details>
+  <summary>Maven top level dependencies. Click me.</summary>
+  Note that opentelemetry-semconv does not exist (yet) as 1.20.1 but only as 1.20.1-alpha.
+  
+  ```
+  
+  <project>
+      <dependencyManagement>
+          <dependencies>
+              <dependency>
+                  <groupId>io.opentelemetry</groupId>
+                  <artifactId>opentelemetry-bom</artifactId>
+                  <version>1.20.1</version>
+                  <type>pom</type>
+                  <scope>import</scope>
+              </dependency>
+          </dependencies>
+      </dependencyManagement>
+      
+      <dependencies>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-api</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-trace</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-metrics</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-common</artifactId>
+          </dependency>
```

Comment:
These three are already imported by `opentelemetry-sdk`
```suggestion
```

Reply:
same reply on top level dependencies to declare